LICENSES/LicenseRef-Slint-commercial.md

```diff
@@ -1,6 +1,394 @@
+Version 1.2, 22 August 2022
 
-Slint is offered under a commercial license. Please contact info@slint-ui.com if you're interested.
+Copyright © 2022 SixtyFPS GmbH <https://slint-ui.com/imprint.html>
 
-See also the [Licensing FAQ](https://github.com/slint-ui/slint/blob/master/FAQ.md#licensing) online
-or find the same information in `FAQ.md` in the top level of your slint repository checkout.
+
+SixtyFPS GmbH with offices at Am Panke-Park 47, 16321 Bernau
+bei Berlin, Germany (“**SixtyFPS**”) grants a license to the software 
+(“**Licensed Software**”) as specified in **Exhibit A** which is provided 
+subject to the terms of this Slint Software License Agreement (the “**Agreement**”). 
+Each Party to this Agreement may be referred to 
+herein individually as a “**Party**” or collectively as the “**Parties**.”
+
+This Agreement incorporates by reference the following Exhibits
+that are attached to this Agreement:
+
+**Exhibit A Licensed Software**
+
+**Exhibit B Support**
+
+Any Exhibit may be amended from time to time by written agreement of the
+Parties.
+
+1.  The Party to whom SixtyFPS grants the Licensed Software under this Agreement 
+    will be referred to herein as the "**Customer**".
+
+2.  As part of the license grant, a Slint Software License Certificate 
+    ("**License Certificate**") will be issued containing details of (1) the
+    license grant, (2) the permitted scope of Customer's use of the Licensed Software
+    and (3) contact details of the Customer. In case of any changes to the information 
+    provided in (2) and (3) above, Customer is obliged to notify SixtyFPS in writing, 
+    not later than 7 days after any such change has occured. 
+
+3.  The Term of this Agreement shall begin on the date mentioned in the License
+    Certificate ("**Effective Date**") and shall continue unless terminated 
+    earlier as herein provided ("**Term**").
+
+4.  The Licensed Software is protected by copyright laws and
+    international copyright treaties, as well as other intellectual
+    property laws and treaties. The Licensed Software is licensed, not
+    sold.
+
+5.  The Licensed Software may provide links to third Party libraries or
+    code (collectively "**Third Party Libraries**") to implement various
+    functions. Third Party Libraries do not comprise part of the
+    Licensed Software. In some cases, access to Third Party Libraries
+    may be included along with the Licensed Software delivery as a
+    convenience for development and testing only. Customer
+    acknowledges (1) that some Third-Party Libraries may require
+    additional licensing of copyright and/or patents from the owners of
+    such, and (2) that distribution of any of the Licensed Software
+    referencing any portion of a Third-Party Library may require
+    appropriate licensing from such third parties. All such Third Party
+    Libraries, along with applicable copyright notices and licenses,
+    will be listed and made available to Customer.
```

Comment:
Could you format this list as properly like done in 17.?

Reply:
done